scss/components/side-nav.scss

```diff
@@ -0,0 +1,136 @@
+@import "./../settings";
+@import "./../mixins";
+@import "./../functions";
+
+/*!
+.fd-side-nav
+    .fd-side-nav__group
+        .fd-side-nav__title
+        .fd-side-nav__list
+            .fd-side-nav__item
+                .fd-side-nav__link (.has-child, .is-active, .is-expanded)
+                .fd-side-nav__sublist
+                    .fd-side-nav__subitem
+                    .fd-side-nav__sublink (.is-active)
+*/
+
+$block: #{$fd-namespace}-side-nav;
+
+.#{$block} {
+
+    //LOCAL VARS
+    $fd-side-nav-text-size: fd-type(1);
+    $fd-side-nav-background-color: fd-color("background", 2);
+    $fd-side-nav-selected-background-color: rgba(fd-color("action", 1), .07);
+    $fd-side-nav-hover-background-color: fd-color("neutral", 1);
+    $fd-side-nav-link-color: fd-color("text", 2);
+    $fd-side-nav-max-width: 250px;
+    $fd-side-nav-link-padding: fd-space(3) fd-space(5);
+    $fd-side-nav-title-padding: fd-space(2) fd-space(5);
+    $fd-side-nav-link-disabled-color: fd-color("text", 3);
+    $fd-side-nav-title-background-color: fd-color("neutral", 1);
+
+
+    //BLOCK BASE *******************************************
+
+    background-color: $fd-side-nav-background-color;
+    min-width: $fd-side-nav-max-width;
+    margin: 0;
+    padding: 0;
+
+    //ELEMENTS *******************************************
+    &__group {
+        list-style: none;
+        margin-bottom: 0;
+    }
+
+    &__title {
+        display: block;
+        margin-bottom: 0;
+        padding: $fd-side-nav-title-padding;
+        @include fd-type("-3", "body", "reg", uppercase);
+        color: $fd-side-nav-link-disabled-color;
+        background:$fd-side-nav-title-background-color;
+
+    }
+
+    &__list,
+    &__sublist {
+        @include fd-reset;
+        display: flex;
+        flex-wrap: wrap;
+        padding-left: 0;
+        margin-bottom: 0;
+        list-style:  none;
+        flex-direction: column;
+    }
+
+    &__sublist{
+        &[aria-hidden="true"] {
+            display: none;
+        }
+    }
+
+    &__link,
+    &__sublink {
+        padding: $fd-side-nav-title-padding;
```

Comment:
OK, so there is a small thing here that I discussed with GD. For the top and bottom padding, 12px is too big, 8px is too small. So I agreed we can do 10px. The thing is that the math still works when you take 10 (top) + 20 (line height) + 10 (bottom) it adds to 40px which still adheres to the system.

So here, just do `$fd-side-nav-title-padding+2px` for the top and bottom padding. It's a bit of a cheat but I'm fine with it.